docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/index.md

```diff
@@ -0,0 +1,8 @@
+---
+type: docs
+title: "Kubernetes Extensions"
+linkTitle: "Kubernetes Extensions"
+weight: 6
+description: >-
+  Kubernetes extensions are add-ons for Kubernetes clusters. The extensions feature on Azure Arc enabled Kubernetes clusters enables usage of Azure Resource Manager based APIs, CLI and portal UX for deployment of extension components (Helm charts in initial release) and will also provide lifecycle management capabilities such as auto/manual extension version upgrades for the extensions. The scenarios in this section will guide on how to use Kubernetes extensions in your Azure Arc enabled Kubernetes cluster in an automated fashion.
+---
```

Comment:
This should be incorporated under the [Unified Operations Use Cases](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/) section and not in its own section. 

I recommend updating the index file for that section.

Reply:
fixed and let me know if there any further updates needed. Thanks!